btk/draw_blends.py

```diff
@@ -417,12 +421,20 @@ def render_blend(self, blend_catalog, psf, filt, survey, seedseq_blend, extra_da
             _blend_image += single_image
 
         # add noise.
-        if self.add_noise:
+        if self.add_noise in ["galaxy", "all"]:
```

Comment:
Doesn't matter too much but these lists you are creating only to check check conditions can be tuples.